fusor-ember-cli/app/routes/deployment.js

```diff
@@ -149,7 +149,15 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
     error: function(reason) {
       console.log(reason);
       var controller = this.controllerFor('deployment');
-      controller.set('errorMsg', reason.responseJSON.error.message);
+      var msg = null;
+      if (reason.responseJSON) {
+        msg = reason.responseJSON.error.message;
+      } else if (reason.message) {
+        msg = reason.message;
+      } else {
+        msg = "Error was reported"
```

Comment:
`reason.responseJSON.error.message` seems to return the `foreman-task` errors, but `reason.message` seems to return the JS errors.
I'm not sure if display "Error was reported" will be helpful. I didn't it when developing, but I thought if will be good feedback from QA for me to know when they encounter the `error` hook on the deployment route (which is the top route that errors bubble up to)